PROJECT

```diff
@@ -1,23 +1,29 @@
 domain: opentelemetry.io
 layout:
-- go.kubebuilder.io/v2
+- go.kubebuilder.io/v3
 multigroup: true
 plugins:
   manifests.sdk.operatorframework.io/v2: {}
 projectName: opentelemetry-operator
 repo: github.com/open-telemetry/opentelemetry-operator
 resources:
-- controller: true
-  group: collector
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: opentelemetry.io
   kind: OpenTelemetryCollector
-  path: github.com/open-telemetry/opentelemetry-operator/apis/collector/v1alpha1
+  path: github.com/open-telemetry/opentelemetry-operator/apis/v1alpha1
```

Comment:
Does this mean that we can only have one version for all CRDs? If we need to bump the one for instrumentation, would we need to bump the one for the collector as well?

Reply:
no, then we change the version in `PROJECT` for a given kind and create `apis/v1` and move the kind there.